### PR TITLE
resulted shell job

### PR DESCRIPTION
**Pull request by cytzrs**
resulted shell job

### CODE REVIEW

**Comment by reugn** [comment]
@cytzrs, could you explain the motivation behind this pull request?

File: quartz/cron_test.go [hunk 1]
```diff
@@ -342,3 +343,106 @@ func TestCronExpressionError(t *testing.T) {
...
+
+var scheduleTable = make(map[string]*CronJob)
+var now = time.Now()
+
+func TestTain(t *testing.T) {
+	// 定时任务字典， key: 任务的名字, value 任务对象
+
+	// 定义定时任务以每 5s 执行一次
+	// MustParse 如果遇到解析 contab 错误时会直接抛出 panic ，不会像 Parse 一样返回一个错误
+	expr := cronexpr.MustParse("*/15 * * * * * *")
```
Comment (rfyiamcool) [inline quartz/cron_test.go:363]: the interval duration is too long.

use assert to validate unit tests. 😁

File: quartz/cron_test.go [hunk 2]
```diff
@@ -342,3 +343,106 @@ func TestCronExpressionError(t *testing.T) {
...
+		},
+	}
+	// 任务注册到调度表
+	scheduleTable[key] = cronJob
+}
+
+func degister(key string) {
+
+	delete(scheduleTable, key)
+	fmt.Println("fsdfsdf")
```
Comment (rfyiamcool) [inline quartz/cron_test.go:447]: 😅 why do u print 'fsdfsdf' here ?

File: test-shell.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+date
```
Comment (rfyiamcool) [inline test-shell.sh:1]: It is recommended to use a temporary directory to generate scripts, or to place files in the testdata directory. 😁

**Pull request closed without merging**